client-lite/build/postrm.in.sh

```diff
@@ -11,29 +11,55 @@ run_path=@docs_svc_run_dir_path@
 svc_name=@docs_svc_name@
 svc_config_path=@docs_systemd_cfg_path@
 
-echo "Running post-removal script for $svc_name"
+echo "**** Running post-removal script for $svc_name, args: $1 $2 ****"
 
-systemctl reset-failed $svc_name # Remove ourselves from the failed services list. No-op if never failed earlier.
+do_remove_daemon() {
+    echo "Removing systemd unit file: $svc_config_path"
+    rm $svc_config_path
```

Comment:
There is no `set -e` in this file but it exists in postrm.in.sh.  Could add that, or avoid pitfalls with that and do explicit error checking via adding `|| return` on the end in function definitions or `|| exit` elsewhere (e.g. `mycmd myargs || return`), which will forward along the non-zero ret code when it fails.

I would encourage running shellcheck on these scripts.

Reply:
I noticed the missing set -e but decided it is out of scope for this PR. I'll make such improvements in a separate PR to avoid introducing more bugs before GA.

Reply:
+1 for shellcheck, thanks